seqr/fixtures/users.json

```diff
@@ -31,6 +31,22 @@
         "permissions": []
     }
 },
+{
+    "model": "auth.group",
+    "pk": 6,
+    "fields": {
+        "name": "subscribers",
+        "permissions": []
+    }
+},
+{
+    "model": "auth.group",
+    "pk": 7,
+    "fields": {
+        "name": "subscribers_external",
```

Comment:
what differentiates a subscriber from an external subscriber?

Reply:
oh it was just a naming choice I made because this group is assigned to a project thats referred to in the tests as "external". In real life every project has its own distinct subscriber group and the memebers can be different, so I wanted to be able to test that the correct project-specific notifications were going to the correct subscribers